test/test_tensor_creation_ops.py

```diff
@@ -1960,6 +1960,54 @@ def test_inference(default_dtype):
         test_inference(torch.float64)
         test_inference(torch.float32)
 
+    @suppress_warnings
+    @onlyCPU
+    def test_tensor_factory_ndarrays(self, device):
+        """
+        This tests the case when the input to torch.tensor are multiple
+        numpy arrays that are embedded (recursively) within list(s). 
+        """
+
+        repeats = 10
+        np_types = [np.int8, np.int16, np.int32, np.int64, np.float16, np.float32,\
```

Comment:
Instead of creating its own NumPy type list this test can reuse the keys in this dict:

https://github.com/pytorch/pytorch/blob/929b91a24d7c78483cf9c0dc434810e640d06a2c/torch/testing/_internal/common_utils.py#L382